pytorch/buildspec-ei.yml

```diff
@@ -8,7 +8,7 @@
     inference_repository: &INFERENCE_REPOSITORY
         image_type: &INFERENCE_IMAGE_TYPE inference
         root: !join [ *FRAMEWORK, "/", *INFERENCE_IMAGE_TYPE ]
-        repository_name: &REPOSITORY_NAME !join [pr, "-", *FRAMEWORK, "-", *INFERENCE_IMAGE_TYPE, "-", eia]
```

Comment:
After this change, Sagemaker test has always failed due to path issue.

Reply:
I am using existing pr repos for eia. I am not sure about the repo name should be either "pr-<framework>-inference-eia" or "pr-framework-eia". 
Are we always testing normal SM tests on EI images?

Reply:
With EI Images, we do not run the local test.
In Remote test, test_ei.py instead of test_tfs.py for TensorFlow inference and few similar change.
Few test like test_gluonnlp and test_gluoncv will be skipped.

Reply:
My point is that it is looking for path like sagemaker_tests/mxnet/eia which does not exist in our repo.
We need to refer the sagemaker_tests/mxnet/integration instead for now or in the future, create a folder sagemaker_tests/mxnet/eia. I am in favor of the first one.

Reply:
If they are sharing some of the same tests with normal inference tests, it is reasonable to manage the test scripts inside the same folder.